apps/aehttp/priv/oas3.yaml

```diff
@@ -1912,6 +1912,34 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  /debug/check-tx/pool/{hash}:
+    get:
+      tags:
+        - internal
+        - debug
+      operationId: GetCheckTxInPool
+      description: "Check if a transaction in the pool can be merged or if it is blocked by something: not enough tokens, missing nonce or something else"
```

Comment:
`merged` is not the right word here... 

Something like "Check if a transaction in the pool is valid in the current state of the block chain..." perhaps?